src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -97,7 +97,6 @@ class ReportActionItemMessageEdit extends React.Component {
      * @param {String} newDraft
      */
     updateDraft(newDraft) {
-        this.textInput.setNativeProps({text: newDraft});
```

Comment:
Let's make sure [this issue](https://github.com/Expensify/App/pull/3510) doesn't come back